internal/cloudapi/v2/v2_disk_test.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 func TestComposeDiskCustomizationsValidation(t *testing.T) {
-	srv, _, _, cancel := newV2Server(t, t.TempDir(), false, false)
+	srv, _, _, cancel := newV2Server(t, t.TempDir(), false, false, false)
```

Comment:
This becomes a bit unwieldy, maybe a good time to switch to a `opts *v2ServerOpts` approach? Here is a diff with what we could potentially do here:
[v2serveropts.diff.txt](https://github.com/user-attachments/files/23257283/v2serveropts.diff.txt)

(could be a followup of course)

Reply:
I considered but got lazy because it's just test stuff.  But you're right :)